### PR TITLE
fix(xmldom) - corrected package and module naming

### PR DESCRIPTION
**Pull request by hgray-instawork**
- Corrected name of module to `@instawork/xmldom`
- Corrected name of package in package-lock to `@instawork/xmldom`

### CODE REVIEW

**Comment by adamstep** [comment]
Note: I haven't figured out why the tests are failing after I forked, the same thing was happening on my PRs

**Review by flochtililoch — COMMENTED** [review]
the version bump should be done via a script instead of manually I assume, @adamstep ?

**Comment by adamstep** [comment]
yes, I did it with a script

**Review by flochtililoch — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@xmldom/xmldom",
+  "name": "@instawork/xmldom",
```
Comment (hgray-instawork) [inline package-lock.json:2]: @adamstep I'm unclear if this should be changed or not

Comment (adamstep) [reply package-lock.json:2]: I think it should be changed as well for consistency.

**Pull request merged**